addons/website/views/snippets/s_contact_info.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_contact_info" name="Contact Info">
+    <section class="s_contact_info pt48 pb64 o_cc o_cc1">
+        <div class="container">
```

Comment:
Check if we can use the `<address>` tag here, it would make a lot of sense ([ref.](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address)).
If the editor doesn’t like it keep `<div>`... it's not a big deal 😉

```suggestion
        <address class="container">
```

Reply:
Kept the div. I think that an `<address>` shouldn't have content like headings and paragraphs inside them.